frontend/src/component/common/VerticalTabs/VerticalTab/VerticalTab.tsx

```diff
@@ -1,25 +1,36 @@
-import { styled } from '@mui/material';
+import { Button, styled } from '@mui/material';
 
-const StyledTab = styled('button')<{ selected: boolean }>(
+const StyledTab = styled(Button)<{ selected: boolean }>(
```

Comment:
Just curious: What was the reasoning behind this change?

Reply:
The first intention was to have the tooltip and the endIcon to the Change Request Configuration tab.... These come built in for `Button`